io-classes/src/Control/Monad/Class/MonadUnique.hs

```diff
@@ -0,0 +1,33 @@
+{-# LANGUAGE TypeFamilies, AllowAmbiguousTypes #-}
```

Comment:
```suggestion
{-# LANGUAGE TypeFamilies           -#}
{-# LANGUAGE TypeFamilyDependencies -#}
```
Explained below.

Reply:
`TypeFamilyDependencies` implies `TypeFamilies`; it's redundant here. But yeah, `AllowAmbiguousTypes` is gone now.

Reply:
You're right, no need for both, but it's fine.